includes/wf_crm_webform_postprocess.inc

```diff
@@ -1992,37 +1992,34 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
       $paymentProcessor->setCancelUrl($this->getIpnRedirectUrl('cancel'));
     }
 
-    if (method_exists($paymentProcessor, 'doTransferCheckout')) {
-      //Paypal Express checkout
-      if(wf_crm_aval($_POST, 'credit_card_number') == 'express'){
-        try {
-          $params['button'] = 'express';
-          $params['component'] = 'contribute';
-          $result = $paymentProcessor->doPreApproval($params);
-            if (empty($result)) {
-            // This could happen, for example, when paypal looks at the button value & decides it is not paypal express.
-            return;
-          }
-        }
-        catch (\Civi\Payment\Exception\PaymentProcessorException $e) {
-          drupal_set_message(ts('Payment approval failed with message :') . $e->getMessage(),'error');
-          CRM_Utils_System::redirect($this->getIpnRedirectUrl('cancel'));
-        }
-        $preApprovalParams = $paymentProcessor->getPreApprovalDetails($result['pre_approval_parameters']);
-        $params = array_merge($params, $preApprovalParams);
-        if (!empty($result['redirect_url'])) {
-          CRM_Utils_System::redirect($result['redirect_url']);
+    //Paypal Express checkout
```

Comment:
There is not point of the code here anymore:
https://github.com/compucorp/webform_civicrm/blob/b0f048e926d3e058d411e10fd1fb9d9f43b653b3/includes/wf_crm_webform_postprocess.inc#L1996-L2015

given it was only called if "doTransferCheckout" exists.